actions/sync-calendar/gcal/testing/test_schedule.yaml

```diff
@@ -0,0 +1,13 @@
+# Generated from: go run actions/generate-schedule/main.go --start 2020-06-01 --stop 2020-07-01 --users abc,lmn,xyz
+shifts:
+- startDate: Mon 01 Jun 2020
```

Comment:
There's no override in any of these shifts; would it be good to test that too?

Reply:
Good idea - adding this I found a bug in my regen_replay script too